### DISCUSSION TITLE
x-show does not work with (nested) x-collapse

### DISCUSSION THREAD

**Discussion by hudri** [alpinejs/alpine#3812] [7. Bugs (archived)]
When using x-show together with (nested) x-collapse, the x-show no longer applies "display: none";

Codepen example:

https://codepen.io/webtourismus/pen/eYbaQXp

**Comment by ekwoka** [comment]
Change your `@click` on 35, 38, 45 to `@click.stop.prevent`

**Comment by hudri** [comment]
Thanks, this fixes my use case. But I think there still is a bug in general with structures like `<div class="parent" x-show><div class="child" x-show x-collapse></div></div>`

Here is a better, more reduced pen, without any @click.outside to interfere: https://codepen.io/webtourismus/pen/gOZJEBz

**Reply by ekwoka** [reply]
It's not nested x-collapses specifically, to clarify, it's closing a nested collapse while a parent collapse is also closing.

Not exactly a hyper common occurrence, but that makes the issue more clear.
